packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -276,7 +281,7 @@ const BlockInspectorSingleBlock = ( {
 						</PanelBody>
 					) }
 
-					{ ! isSectionBlock && (
+					{ ! isSectionBlock && ! isNavigationMode && (
```

Comment:
Are these tests the same? When could they be different?

Here's the selector:

https://github.com/WordPress/gutenberg/blob/a2662dfcc38a3fe6757eb4ec6a307b6f709d13ce/packages/block-editor/src/store/private-selectors.js#L519-L519